### DISCUSSION TITLE
Strange error

### DISCUSSION THREAD

**Discussion by Vadimluck** [openai/whisper#1546] [Q&A] [answered]
Strange error
![nn](https://github.com/openai/whisper/assets/134635778/e98508e5-ec0c-4575-87cc-4131b872b99e)
Can you please tell me what I can do to fix it?
ffmpeg -version is working.
![1](https://github.com/openai/whisper/assets/134635778/6ef47f6a-7244-4cc9-bac3-eea6245eeea4)
everything was working, but after reinstalling python it stopped.

**Comment by Vadimluck** [comment, marked as answer]
I dealt with the error by just removing the extra python. I had python39 and python310, maybe whisper is not compatible with python310, but after removing python310 everything worked as if the problem didn't exist at all.

**Comment by Sudhanshu2-0** [comment]
<img width="676" alt="image" src="https://github.com/openai/whisper/assets/140401081/9b7615da-f895-4d16-a1cb-4e714196883f">
I'm also getting this error, what should i do?
